tests/test_leaderboard.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+import pandas as pd
+import pytest
+
+from mteb.benchmarks import MTEB_EN, MTEB_ENG_CLASSIC, Benchmark, MTEB_multilingual
+from mteb.leaderboard.table import scores_to_tables
+from mteb.leaderboard.utils import filter_models, load_results
+
+# A couple of models to look for.
+SHOULD_BE_THERE = [
+    "NV-Embed-v2",
+    "e5-mistral-7b-instruct",
+    "text-embedding-005",
+    "voyage-large-2-instruct",
+    "cde-small-v2",
+    "SONAR",
+    "text-embedding-3-small",
+    "GritLM-7B",
+    "sentence-t5-xxl",
+    "M2V_base_glove",
+]
+BENCHMARKS_TO_CHECK = [
+    MTEB_EN,
+    MTEB_ENG_CLASSIC,
+    MTEB_multilingual,
+]
+
+all_results = load_results()
```

Comment:
This should be moved to a fixture (otherwise it is loaded even on `pytest --collect-only`

Reply:
We load just the target model and tasks to speed this up.